public/locales/en.json

```diff
@@ -70,6 +70,12 @@
                     "textWithFile": "While trying to import history from '{{fileName}}' an error occurred.",
                     "textWithoutFile": "You did not provide any files to upload the history from.",
                     "tooNewVersion": "The file '{{fileName}}' comes from a newer client and can't be imported."
+                },
+                "removeNote": {
+                  "title": "Remove note from history",
+                  "question": "Do you really want to remove this note from your history?",
```

Comment:
Do we maybe include the name of the note here?

Reply:
Done